docs/source/notebooks/Tutorial.ipynb

```diff
@@ -1879,7 +1879,7 @@
     "- We can use additional photons called ancillas, which we can measure independently from the main circuit photons. Depending on the state obtained on the ancilla, we know whether the gate has succeeded or not on the main qubits. Those gates will be called heralded.\n",
     "- We can also directly measure the main circuit qubits, and depending on the result, assess whether the gate has succeeded or not. Those gates will be called postselected.\n",
     "\n",
-    "The CNOT gate acts on two qubits, a control and a target, and flips the value of the target if the control qubit is in state $|1\\rangle_L$. In the following two exercices, we will see the two types of CNOT gates: \n",
+    "The CNOT gate acts on two qubits, a control and a target, and flips the value of the target if the control qubit is in state $|1\\rangle_L$. In the following two exercise, we will see the two types of CNOT gates: \n",
```

Comment:
we need plural "exercises"

Reply:
done https://github.com/Quandela/Perceval/pull/435/commits/3c9d20e9e1a188ae01afb445c36ff3d609ad841a